draft-ietf-mls-protocol.md

```diff
@@ -4854,45 +4854,63 @@ outside the group:
 * Group ID
 * Group membership
 * Group extensions
-* Frequency of epoch changes
+* Current epoch and frequency of epoch changes
 * Frequency of application messages within an epoch
 
+### DS Architecture
+
+The following is relevant for applications where the DS is an entity less
+trusted than the members’ clients (e.g. a serverside component).
```

Comment:
The DS is always less trusted than the members clients.  The whole point of MLS is to protect against the DS :)  If you're thinking of things like p2p environments, then the DS would be the p2p links and any overlay routing, which is still less trusted than the client itself.

Reply:
This has lead to quite a bit of confusion in the past and the architecture document is a bit ambiguous in the way the DS is defined. On the one hand you're right: The role of the DS involves message routing and it's described as a less trusted component. On the other hand, the role of the DS also involves commit ordering, which in case of a p2p system happens on the clients and which can not be done by overlay routing or p2p links.

I do agree though, that even as an abstract component, we can generally say that the DS is considered less trusted, even if it's (at least partially) running within the trust boundary of the MLS client.